frontend/src/views/MyPermissionsView.vue

```diff
@@ -0,0 +1,7 @@
+<script setup lang="ts">
+import MyPermissions from '@/components/myPermissions/MyPermissions.vue';
+</script>
+
+<template>
+    <MyPermissions></MyPermissions>
```

Comment:
This is small, but if we don't pass properties and has no child elements, it can just be  <MyPermissions />

Reply:
Sure, I'll change it to self closing